P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -2608,6 +2608,33 @@ effective than the free form <gi>note</gi> element; these elements are
 available when the module for simple analysis is selected (see section
 <ptr target="#AISP"/>).
  </p>
+
+ <p>When alternative versions of the same note, e.g. in different languages or
```

Comment:
Sentence needs to be rewritten, possibly as multiple sentences. (It boils down to “When alternative versions of the same note … it is possible to group such notes within …”.)

Perhaps something like
~~~xml
A text may have multiple alternative versions of the same note. For
example the same annotation expressed in multiple languages, or
both an extensive note and a short form for different audiences. In
such cases multiple <gi>note</gi> elements can be grouped within a
<gi>noteGrp</gi> element.
~~~